examples/netlify-edge/build.js

```diff
@@ -0,0 +1,20 @@
+const { build } = require('esbuild')
+
+async function main() {
+  await build({
```

Comment:
It looks good to me but why do we need a build script here?

Reply:
Netfliy edge does it’s own thing ad wasn’t working when I imported a node module. It liked loading from CI. Instead I made it so we bundle everything and not have to deal with inconsistencies